cartography/data/indexes.cypher

```diff
@@ -129,6 +129,9 @@ CREATE INDEX ON :GSuiteGroup(email);
 CREATE INDEX ON :GSuiteGroup(id);
 CREATE INDEX ON :GSuiteUser(email);
 CREATE INDEX ON :GSuiteUser(id);
+CREATE INDEX ON :GCPFunction(id);
```

Comment:
these 3 indexes already exist. we duplicated them by mistake. let's remove these 3 lines.